### DISCUSSION TITLE
Failed to push to the remote Repository.

### DISCUSSION THREAD

**Discussion by JjayFabor** [community/community#54942] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hello, I need help!

I've been trying to push the updated code in my github repository but it gives this error "Failed to push to the remote repository". So I tried creating a new branch to push the code but it gives me the same error. 

Thank you in advance.

**Comment by iamreiyn** [comment, marked as answer]
Hi @JjayFabor 

There's an ongoing incident and major outage with several GitHub services, check [GitHub Status](https://www.githubstatus.com/) to read more, thank you.

**Reply by JjayFabor** [reply]
Thank you. It already pushes to the repository.
